src/util/builders/url/base.director.ts

```diff
@@ -10,7 +10,13 @@ export default class BaseURLDirector {
     this.baseDomain = baseDomain
   }
 
-  buildURL(path: string) {
+  buildURL(path: string, wikiPath = '') {
```

Comment:
command line inputs should be sanitize in `sanitize-argument.ts`

Reply:
Done.